src/neo/SmartContract/Native/NativeContract.cs

```diff
@@ -17,6 +17,7 @@ public abstract class NativeContract
         private static readonly Dictionary<string, NativeContract> contractsNameDictionary = new Dictionary<string, NativeContract>();
         private static readonly Dictionary<UInt160, NativeContract> contractsHashDictionary = new Dictionary<UInt160, NativeContract>();
         private readonly Dictionary<string, ContractMethodMetadata> methods = new Dictionary<string, ContractMethodMetadata>();
+        public static readonly Dictionary<uint, List<NativeContract>> ActiveBlockIndexes = new Dictionary<uint, List<NativeContract>>();
```

Comment:
Move it to `ManagementContract`?

Reply:
Then, set `ActiveBlockIndex` must be internal, it's ok for you?

Reply:
Currently we have 6 native contracts only. I think traversal is faster than hash table in this case.